### PR TITLE
el6 PEM fix

### PR DESCRIPTION
**Pull request by punkrokk**
This PR updates the ssh-keygen PEM arg not present in EL6

### CODE REVIEW

File: scripts/includes/common.sh [hunk 1]
```diff
@@ -142,7 +142,12 @@ configure_st2_user () {
   # to be generated again.
   if ! sudo test -s ${SYSTEM_HOME}/.ssh/stanley_rsa; then
     # added PEM to enforce PEM ssh key type in EL8 to maintain consistency
-    PEM="-m PEM"
+    if [[ "$RHMAJVER" != '6' ]]; then
+        PEM="-m PEM"
+    else
+        # PEM flag not present in ssh-keygen version in EL6
+        PEM=""
```
Comment (armab) [inline scripts/includes/common.sh]: Please use consistent indentation: 2 spaces instead of 4, following the existing code.

Comment (punkrokk) [reply scripts/includes/common.sh]: fixed

File: scripts/st2bootstrap-el8.sh [hunk 2]
```diff
@@ -257,7 +257,12 @@ configure_st2_user () {
   # to be generated again.
   if ! sudo test -s ${SYSTEM_HOME}/.ssh/stanley_rsa; then
     # added PEM to enforce PEM ssh key type in EL8 to maintain consistency
-    PEM="-m PEM"
+    if [[ "$RHMAJVER" != '6' ]]; then
```
Comment (armab) [inline scripts/st2bootstrap-el8.sh]: `RHMAJVER` is not available if script is ran directly as `st2bootstrap-el8.sh`. That's an issue.

Comment (punkrokk) [reply scripts/st2bootstrap-el8.sh]: @armab I think I resolved this: https://github.com/StackStorm/st2-packages/pull/637/files

**Pull request merged**